plugin.video.puls4/resources/lib/xxtea.py

```diff
@@ -0,0 +1,287 @@
+#/**********************************************************\
```

Comment:
This would be better in a dependency/script.module. Since it is small I'm fine with keeping with with the addon.